app/src/main/res/color/navigation_item.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/colorPrimary" android:state_selected="true" />
+    <item android:color="@color/lighter_grey" android:state_selected="false" />
+</selector>
```

Comment:
I think it can be add to `styles.xml`.

Reply:
I put it there because the documentation at [developer.android.com](https://developer.android.com/guide/topics/resources/color-list-resource) said color state list resources should be put in the color directory.